whisper-script/reference-transcripts.py

```diff
@@ -0,0 +1,42 @@
+import whisper
+import os
+
+model = whisper.load_model("medium")
+
+#list audio files in current folder
+
+files=os.listdir('.')
+# Select the language of the Whisper model
+auto_detect = True # Set to True if you do not know the language of the audio
+default_lang = 'en' # If auto_detect is True, this value is ignored
+
+#Process each audio files in whisper, wirting in reference_transcriptions_whisper.csv
+with open('./reference_transcriptions_whisper.csv','w', encoding='utf-8') as csvfile:
+    #Writing CSV column headers
+    csvfile.write('Language - CAN BE DELETED,Audio File Name,Reference')
```

Comment:
I'd prefer not to add a manual step if possible.  I trust your judgement.

Reply:
agreed - working on it